lib/empty-example/sketch.js

```diff
@@ -1,7 +0,0 @@
-function setup() {
```

Comment:
Was this intentionally deleted? Otherwise I think we can keep this file around

Reply:
Ooh... Sorry